meteor/client/ui/Settings/ShowStyleBaseSettings.tsx

```diff
@@ -78,9 +78,16 @@ export default translateWithTracker<IProps, IState, ITrackedProps>((props: IProp
 	return {
 		showStyleBase: showStyleBase,
 		showStyleVariants: showStyleBase
-			? ShowStyleVariants.find({
-					showStyleBaseId: showStyleBase._id,
-			  }).fetch()
+			? ShowStyleVariants.find(
+					{
+						showStyleBaseId: showStyleBase._id,
+					},
+					{
+						sort: {
+							_rank: 1,
```

Comment:
Since it currently is possible in `importShowStyleVariant` to add variants with duplicate `_rank`-values, we need to add a secondary argument for sorting:
```
sort: {
  _rank: 1,
  _id: 1,
}
```